meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -477,6 +477,7 @@ PREFERRED_PROVIDER_virtual/bootloader:stm32mp1common ?= "u-boot-fio"
 PREFERRED_PROVIDER_u-boot:stm32mp1common ?= "u-boot-fio"
 PREFERRED_PROVIDER_virtual/trusted-firmware-a:stm32mp1common = "tf-a-fio"
 MACHINE_EXTRA_RRECOMMENDS:remove:stm32mp1common = "u-boot-fw-config-stm32mp"
+MACHINE_EXTRA_RRECOMMENDS:append:stm32mp1common = "linux-firmware-bcm43430"
```

Comment:
misses the space `MACHINE_EXTRA_RRECOMMENDS:append:stm32mp1common = " linux-firmware-bcm43430"`